src/ape/utils/trace.py

```diff
@@ -292,22 +292,27 @@ def parse_as_gas_report(self, call: CallTreeNode) -> List[Table]:
 
         for contract_id, method_calls in report.items():
             title = f"{contract_id} Gas"
-            table = Table(title=title, box=BOX_TYPE)
+            table = Table(title=title, box=SIMPLE)
             table.add_column("Method")
             table.add_column("Times called")
-            table.add_column("Min.")
-            table.add_column("Max.")
-            table.add_column("Mean")
-            table.add_column("Median")
+            table.add_column("Min.", justify="right")
+            table.add_column("Max.", justify="right")
+            table.add_column("Mean", justify="right")
+            table.add_column("Median", justify="right")
 
             for method_call, gases in method_calls.items():
+
+                def _format(val: Any) -> str:
+                    # Add commas if > 6 length
```

Comment:
since the gas values tend to be of smaller magnitudes, maybe drop the thousands commas in all cases.